mlir/lib/Dialect/SparseTensor/Transforms/Sparsification.cpp

```diff
@@ -1064,6 +1067,10 @@ static bool startLoopSeq(CodegenEnv &env, OpBuilder &builder, ExprId exp,
 
   SmallVector<TensorLevel> tidLvls;
   getAllTidLvlsInLatPoints(env, l0, curr, [&](TensorLevel tl, AffineExpr) {
+    // TODO: remove this! Duplication can be introduced due to the speical
```

Comment:
typo: special

also, I don't quite understand the "can be introduced" part of this?